src/Componenets/Categories.jsx

```diff
@@ -24,10 +24,10 @@ const Categories = () => {
         value={selectedCategory}
         onChange={(event) => setSelectedCategory(event.target.value)}
       >
-        <option value="">Select a Category</option>
+        <option value="">All Reviews</option>
         {categories.map((category, index) => {
           return (
-            <option key={index} value={category}>
+            <option key={index} value={category.slug}>
               {category.slug}
```

Comment:
It's good that you're populating this list of categories using an API request for the categories 👍 
It could be made even better by showing the user a formatted list, which for example shows "Hidden roles", rather than "hidden-roles" in the dropdown menu.